shark_turbine/kernel/wave/expansion.py

```diff
@@ -293,6 +297,7 @@ def expand_graph(
                 node_index_setter,
                 expansion_context,
             )
+    remove_chained_getresult(trace)
```

Comment:
Should we move this out of this function to after expansion? You can move it to wave.py. This way we can isolate what expansion is doing vs the others.

Reply:
makes sense, done!